### ISSUE TITLE
"File already exists. Do you want to overwrite it?" is shown twice

### ISSUE THREAD

**Issue by coldspark29** [xournalpp/xournalpp#998]
If you want to save a file with a name, that already exists, you get two notifications. One notification from GTK3 I guess and the other from Xournal++. Might also be that it has to do with me being on KDE. There is also a spelling error in the German notification btw

**Comment by peetCreative** [comment]
@luco5826 has fixed things for extension in #594 .
I would like to have the gtk dialog, because it's much more consistent.

**Comment by andreasb242** [comment]
The GTK Dialog does not allow to select a file extension, therefore e.g. Gimp also uses a Custom export handling like this...

**Comment by peetCreative** [comment]
According to Gimp: They also use the GTK overwrite dialog, but handle it using signals.
https://gitlab.gnome.org/GNOME/gimp/blob/master/app/dialogs/file-save-dialog.c

**Comment by peetCreative** [comment]
I think one could implement export, save, open dialogs similar to gimp, using function connected to signals.

**Comment by JJones780** [comment]
It looks like GIMP plays some tricks to have the dialog do the overwrite check on a modified filename. 
I tested the "response" signal but that occurs AFTER the overwrite warning which is why GIMP runs through the response twice, the second time around is with the modified filename and overwritewarning turned on . It seems like a bit too much hassle. Didn't GTK originate with GIMP.. why are they hacking a solution?  Maybe I'm wrong in how they are doing it.
I also looked into using the "update-preview"  signal... which happens on every little change but I think that's asking for trouble.
